test/language/statements/for-in/S12.6.4_A7_T2.js

```diff
@@ -25,23 +28,7 @@ for (var __key in __obj){
 	
 }
 
-
-//////////////////////////////////////////////////////////////////////////////
-//CHECK#1
-if (!((__accum.indexOf("aa1")!==-1)&&(__accum.indexOf("ca3")!==-1))) {
-	throw new Test262Error('#1: (__accum.indexOf("aa1")!==-1)&&(__accum.indexOf("ca3")!==-1)');
-}
-//
-//////////////////////////////////////////////////////////////////////////////
-
-//////////////////////////////////////////////////////////////////////////////
-//CHECK#2
-if (__accum.indexOf("ba2")!==-1) {
-	throw new Test262Error('#2: __accum.indexOf("ba2") === -1. Actual:  __accum.indexOf("ba2") ==='+ __accum.indexOf("ba2")  );
-}
-//
-//////////////////////////////////////////////////////////////////////////////
-
+assert.sameValue(__accum, "aa1ca3");
```

Comment:
The specification does not guarantee the iteration order of `for-in` loops, so I think this can possibly fail in valid implementations.

Reply:
Ah, right. Thanks